src/humidity.rs

```diff
@@ -0,0 +1,242 @@
+//! Types and constants for handling humidity.
+
+use super::measurement::*;
+use density::Density;
+use pressure::Pressure;
+use temperature::Temperature;
+
+/// The `Humidity` struct can be used to deal with relative humidity
+/// in air in a common way. Relative humidity is an important metric used
+/// in weather forecasts.
+///
+/// Relative humidity (as a ratio and percentage) and conversions between
+/// relative humidity and dewpoint are supported. It also provides calculations
+/// giving vapour pressure and absolute humidity.
+///
+/// Relative humidity gives the ratio of how much moisture the air is
+/// holding to how much moisture it could hold at a given temperature.
+/// Here we use the technical definition of humidity as ratio of the
+/// actual water vapor pressure to the equilibrium vapor pressure
+/// (often called the "saturation" vapor pressure).
+///
+/// For dewpoint calculations, we use the algorithm commonly known as
+/// the Magnus formula, with coefficients derived by Alduchov and
+/// Eskridge (1996), which gives resonable accuracy (vapour pressure
+/// error < 0.2%) for temperatures between 0 deg C, and 50 deg C.
+///
+/// # Example:
+///
+/// ```
+///     //  calculate the dewpoint from the relative humidity
+///     use measurements::{Humidity,Temperature};
+///
+///     let humidity = Humidity::from_percent(85.0);
+///     let temp = Temperature::from_celsius(18.0);
+///     let dewpoint = humidity.as_dewpoint(temp);
+///     println!("At {} humidity, air at {} has a dewpoint of {}", humidity, temp, dewpoint);
+///
+/// ```
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[derive(Copy, Clone, Debug)]
+pub struct Humidity {
+    relative_humidity: f64, // expressed as a percentage
+}
+
+impl Humidity {
+    /// Create a new Humidity from a floating point value percentage (i.e. 0.0% to 100.0%)
+    pub fn from_percent(percent: f64) -> Self {
```

Comment:
percent is right, I feel. 
I checked with a google search and found widespread consensus online. These are typical: 
"The rule for using percent and percentage is straightforward. The word percent (or the symbol %) accompanies a specific number, whereas the more general word percentage is used without a number."
and, 
"Percentage" is used to refer to a general relationship rather than a specific measure: A large percentage of the people voted, but only 20 percent of the votes counted".

So I vote for percent.